app/services/idv/steps/link_sent_step.rb

```diff
@@ -28,10 +28,9 @@ def check_if_take_photo_with_phone_successful
 
```

Comment:
This step should always be marked complete.  The hybrid flow will complete the step regardless of if it's disabled or not